descriptors/ModuleDescriptor-template.json

```diff
@@ -257,6 +257,18 @@
           ],
           "modulePermissions": [
           ]
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/data-export/fieldNames",
```

Comment:
I think we give a better name that is more readable, like transformations or mappingTransformations

Reply:
I can rename it here, but in the code, we have transformations already...so that`s why I choose field names